fhttp/http_test.go

```diff
@@ -705,3 +705,25 @@ func BenchmarkFoldFind(b *testing.B) {
 		FoldFind(testHaystack, needle)
 	}
 }
+
+func TestChangeMaxPayloadSize(t *testing.T) {
+	var tests = []struct {
+		input    int
+		expected int
+	}{
+		// negative test cases
+		{-1, 0},
+		// lesser than current default
+		{0, 0},
```

Comment:
(semi optional) add a test (spinning an echo server like other tests in this file) making such request, does it work with 0 and with 1 ?

[we may get race test failure on changing the size btw... might have to use the non race file to test that]

Reply:
it works for 0 and 1. Tested it manually.  (semi optional) => Next PR